transformer_engine/pytorch/attention.py

```diff
@@ -5575,14 +5581,26 @@ def forward(
                     qkv_dtype,
                 ).view(out_fp8.shape)
 
-            fp8_tensors = (
-                q_fp8,
-                k_fp8,
-                v_fp8,
-                out_fp8,
-                fp8_meta["scaling_fwd"].scale.clone(),
-                fp8_meta["scaling_fwd"].scale_inv.clone(),
-            )
+            if not int(os.getenv("NVTE_FP8_DPA_BWD", "1")):
+                fp8_tensors = (None, None, None, None, None, None)
+            # elif fp8_meta["recipe"].fp8_mha:
+            #    fp8_tensors = (
+            #        None,
+            #        None,
+            #        None,
+            #        None,
+            #        fp8_meta["scaling_fwd"].scale.clone(),
+            #        fp8_meta["scaling_fwd"].scale_inv.clone(),
+            #        )
```

Comment:
Debugging code?
```suggestion
```
Also, why does the unfused case have different logic for `fp8_meta["recipe"].fp8_mha` than the QKV-fused and KV-fused cases?